app/controllers/applications_controller.rb

```diff
@@ -17,7 +17,7 @@ def create
     @application = Application.new(application_params)
     @application.event = @event
     if @application.save
-      redirect_to event_application_path(@event, @application)
+      redirect_to @event
```

Comment:
You might want to look into flash messages here – if the user gets redirected to the event page, it would be nice to explain why: http://guides.rubyonrails.org/action_controller_overview.html#the-flash